contracts/contribute/ContributeManager.sol

```diff
@@ -195,13 +201,19 @@ contract ContributeManager {
     /// @param socialId Contributor social Id.
     /// @param stakingInstance Contribute staking instance address.
     function createAndStake(uint256 socialId, address stakingInstance) external payable {
+        // Reentrancy guard
```

Comment:
Added reentrancy guards all the way through due to external function calls, even though we trust them